docs/usage/cli/filter.rst

```diff
@@ -63,3 +71,184 @@ The filter command allows you to select a specific number of sequences from spec
     --output filtered.fasta
 
 This subsampling and filtering will reduce the number of sequences in the tutorial data set from 34 to 24.
+
+Subsampling beyond the capabilities of ``augur filter``
+-------------------------------------------------------
```

Comment:
(Discussion on placement of docs)

Reply:
While this doc section is nested deep within docs.nextstrain.org, I think it's a good place since it allows cross-referencing of `augur filter`options, and fits nicely right after the guide on subsampling within `augur filter`.

Reply:
I like this placement for now, since it keeps the examples close to the CLI doc for augur filter.

I'd consider a different name for this section that gives a clearer description of what the contents will be. You could use "calling augur filter multiple times" here instead or "tiered subsampling" or "hierarchical subsampling" or "combining multiple independent filter outputs".

Reply:
I think it makes sense to keep close to the augur filter docs. 

We could add a link to the "Guides" section in the top level docs.nextstrain.org [Bioinformatics/TOC](https://github.com/nextstrain/docs.nextstrain.org/blob/2ca0b7266ccdb86bf27dd91f774071fdc133ea0e/src/guides/bioinformatics/index.rst) so it's more accessible.

Reply:
@huddlej I'm still in favor of the title "Subsampling beyond the capabilities of augur filter" because it conveys _when_ the user would want to look at this section, and describes _why_ multiple calls are necessary. As the capabilities of `augur filter` are expanded, there will be scenarios that will no longer require multiple calls.

To complete the picture here, the current "subsampling" section should highlight the capabilities and limitations of subsampling within `augur filter`. I can add that in this PR.

Reply:
I spoke with @huddlej in person about this. Updated the two section names in 3f58fc32:

> Subsampling within `augur filter`

> Subsampling using multiple `augur filter` commands

Reply:
I've added an entry to the PR checklist:

> Post-merge: Link to these docs from `docs/guides/bioinformatics`.

Marking this as resolved.